examples/stm32f4xx/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "usb-audio"
+version = "0.1.0"
+authors = ["Kiffie van Haash", "Josef Utbult"]
+license = "MIT"
```

Comment:
Could you please replace with ""MIT OR Apache-2.0"?

Reply:
Sure! I am not sure to what you are referring to with ISO endpoints, but everything seems to be working as far as I understand

Reply:
Ok. So we have a working stm32 example, which is great.